### PR TITLE
Fix for debian 10+

### PR DESCRIPTION
**Pull request by Razuuu**

### CODE REVIEW

File: wireguard-install.sh [hunk 1]
```diff
@@ -123,13 +123,17 @@ function installWireGuard() {
 		apt-get update
 		apt-get install -y wireguard iptables resolvconf qrencode
 	elif [[ ${OS} == 'debian' ]]; then
-		if ! grep -rqs "^deb .* buster-backports" /etc/apt/; then
+		if [[ ${PRETTY_NAME} == "Debian GNU/Linux 10 (buster)" ]]; then
+			if ! grep -rqs "^deb .* buster-backports" /etc/apt/; then
 			echo "deb http://deb.debian.org/debian buster-backports main" >/etc/apt/sources.list.d/backports.list
-			apt-get update
-		fi
+			apt update
```
Comment (jaminmc) [inline wireguard-install.sh:129]:
```suggestion
			apt-get update
```
Scripts are supposed to use apt-get not apt.

File: wireguard-install.sh [hunk 2]
```diff
@@ -123,13 +123,17 @@ function installWireGuard() {
...
+			fi
 		apt update
-		apt-get install -y iptables resolvconf qrencode
-		apt-get install -y -t buster-backports wireguard
+		apt install -y -t buster-backports wireguard
+			else
+			apt install -y wireguard
+		fi
+		apt install -y iptables resolvconf qrencode
```
Comment (jaminmc) [inline wireguard-install.sh:136]:
```suggestion
		apt-get install -y -t buster-backports wireguard
			else
			apt-get install -y wireguard
		fi
		apt-get install -y iptables resolvconf qrencode
```
apt-get, not apt

**Review by jaminmc — COMMENTED** [review]
Scripts need `apt-get`, not `apt`.

**Comment by Razuuu** [comment]
sry, my bad

**Comment by Razuuu** [comment]
bro this guy do not read pull requests

**Pull request closed without merging**